src/sbvr-api/sbvr-utils.coffee

```diff
@@ -662,6 +661,13 @@ exports.getAbstractSqlModel = getAbstractSqlModel = (request) ->
 	request.abstractSqlModel ?= abstractSqlModels[request.vocabulary]
 	return request.abstractSqlModel
 
+cloneReq = (req, odata) ->
+	req.method = odata.method
```

Comment:
This modifies the original `req` object which means the logging will show the request as being the first of the batch requests rather than as being a batch request